codecov.yml

```diff
@@ -0,0 +1,4 @@
+ignore:
+  - "src/accessibilityIndex.jsx"
```

Comment:
Should we even collect coverage for these files? If not, we could add them to `coveragePathIgnorePatterns` in package.json, and then coverage won't be reported when tests are run locally as well as on codecov.

Reply:
Cool, TIL. Thanks!

Reply:
Nit: I'm not sure if we need this file anymore because you added it to package.json, but if you don't feel like rebasing again, it's probably fine to leave.